stroyprombeton/management/commands/parse_stalbeton.py

```diff
@@ -9,88 +10,119 @@
 
+    @property
+    def h1(self) -> str:
+        return self.soup.find('h1').text
 
-class CatalogPage(Page):
-    path = 'catalog'
+    @property
+    def description(self) -> str:
+        return self.soup.select_one('meta[name="Description"]')['content']
 
-    def roots(self) -> typing.List['RootCategoryPage']:
-        # parse root categories
-        pass
 
-    def second_level(self) -> typing.List['SecondLevelCategoryPage']:
-        # parse second level categories
-        pass
+class CategoryPage(Page):
+    @property
+    def text(self) -> str:
+        """
+        Only category page has unique text.
+
+        Every another text has autogenerated content.
+        """
+        return self.soup.select_one('#js-category-description').text
 
 
 class RootCategoryPage(CategoryPage):
-    pass
+    def second_level(self) -> typing.List['SecondLevelCategoryPage']:
+        return [
+            SecondLevelCategoryPage(p['href'])
+            for p in self.soup.select('h2 > a.catalog-list__link')
+        ]
 
 
 class SecondLevelCategoryPage(CategoryPage):
     def third_level(self) -> typing.List['ThirdLevelCategoryPage']:
-        # parse it
-        pass
+        return [
+            ThirdLevelCategoryPage(p['href'])
+            for p in self.soup.select('h2 > a.catalog-list__link')
+        ]
 
 
 class ThirdLevelCategoryPage(CategoryPage):
     def options(self) -> list:
-        # parse it
-        pass
+        # @todo #741:60m  Parse stalbeton's options.
+        raise NotImplemented()
+
+    # @todo #741:60m Parse series from stalbeton.
+    #  Series are already parsed as text strings.
+    #  Parse them as separated pages to get options-series relation.
+    def series(self) -> typing.List[str]:
+        return [
+            item.text for item in self.soup.select(
+                '.documentation-block span.documentation-block__item > a'
+            )
+        ]
 
 
-# @todo #736:120m  Parse stalbeton pages.
-#  See "parse it" comments inside the classes.
 def parse():
-    # @todo #736:60m  Try to use stalbeton sitemap.xml.
-    #  To get all category links. If you'll be succeed, use it
-    #  instead of nested category parsing, drafted with the code below.
-    roots = CatalogPage().roots()  # Ignore PyFlakesBear
-    second_level = CatalogPage().second_level()
-    third_level = chain.from_iterable((s.third_level() for s in second_level))
-    options = chain.from_iterable((t.options() for t in third_level))  # Ignore PyFlakesBear
-    # save it into our DB
+    main = Page(path='/')
+    through = ThroughElements(page=main)
+    roots = through.roots()
+    seconds = chain.from_iterable((r.second_level() for r in roots))
+    thirds = chain.from_iterable((s.third_level() for s in seconds))
+    options = chain.from_iterable((t.options() for t in thirds))  # Ignore PyFlakesBear
+    # @todo #741:60m Save parsed stalbeton to a DB.
+    #  It should not some high performance DB.
```

Comment:
Why not? We already have dockerized postgres, that we don't need to setup, so we can just create a separate database for these data.

With sqlite we have to manage two difference databases in the project

I'd prefer to stay with postgres. Up to you

Reply:
@artemiy312 , we'll make this decision during the substask implementation.
I've meant that DB is not required to be high performance. I'll clarify tasks body with it